agreement/proposalTracker.go

```diff
@@ -165,6 +165,11 @@ func (t *proposalTracker) handle(r routerHandle, p player, e event) event {
 		t.Freezer = t.Freezer.freeze()
 		return e
 
+	case readLowestVote:
+		e := e.(readLowestEvent)
+		e.Vote = t.Freezer.Lowest
```

Comment:
What happens if `t.Freezer.Filled == false`?  Can this happen if, for example, this node never saw the proposal, for whatever reason, and just sees certificate votes approving the proposal?

Reply:
Good point! I fixed it now.